docs/src/components/HomepageFeatures/index.tsx

```diff
@@ -0,0 +1,51 @@
+import type {ReactNode} from 'react';
+import clsx from 'clsx';
+import Heading from '@theme/Heading';
+import styles from './styles.module.css';
+
+type FeatureItem = {
+  title: string;
+  Svg: React.ComponentType<React.ComponentProps<'svg'>>;
+  description: ReactNode;
+};
+
+const FeatureList: FeatureItem[] = [
+  // {
+  //   title: 'Easy to Use',
+  //   Svg: require('@site/static/img/undraw_docusaurus_mountain.svg').default,
+  //   description: (
+  //     <>
+  //       Docusaurus was designed from the ground up to be easily installed and
+  //       used to get your website up and running quickly.
+  //     </>
+  //   ),
+  // },
+];
+
+function Feature({title, Svg, description}: FeatureItem) {
+  return (
+    <div className={clsx('col col--4')}>
+      <div className="text--center">
```

Comment:
It's not clear to me where these class names are coming from, is there a CSS framework available?